package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google/model-viewer",
-  "version": "0.0.7",
+  "version": "0.1.0",
```

Comment:
Wonder why 5ad748cfd818ecfc4076809509fe5286b72b5eb7 didn't pick this up? Something to scope out for next release